sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -2895,6 +2912,10 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
 
   lazy val avroDebugDumpAlways: Boolean = get(AVRO_DEBUG_DUMP_ALWAYS)
 
+  lazy val useHybridParquetReader: Boolean = get(HYBRID_PARQUET_READER)
+
+  lazy val loadHybridBackend: Boolean = get(LOAD_HYBRID_BACKEND)
```

Comment:
Right now, it's only used like it must be true if useHybridParquetReader is true.
Where is the code to check this config then load the backend?

Reply:
`LOAD_HYBRID_BACKEND` is a startup config, while `HYBRID_PARQUET_READER` is not. User can config `LOAD_HYBRID_BACKEND` as true on the startup time, and enable/disable `HYBRID_PARQUET_READER` at runtime on the fly.  This is more flexible.

Reply:
Should we have some code the check LOAD_HYBRID_BACKEND then try to load the jar when initializing the driver and executor plugin?

Reply:
It's in Hybrid execution repo. It's like:
HybridPluginWrapper.java:
```java
HybridPluginWrapper {
  DriverPluginWrapper {
     if (LOAD_HYBRID_BACKEND) {
        load_impl()
     }
  }

  ExecutorPluginWrapper {
     if (LOAD_HYBRID_BACKEND) {
        load_impl()
     }
  }
}
```

Also Hybrid execution repo provides a config file `spark-rapids-extra-plugins`:
```
com.nvidia.spark.rapids.hybrid.HybridPluginWrapper
```  
If the Hybrid jar in the classpach, Rapids Plugin uses a reflection approach to load the  `HybridPluginWrapper` and init it. Of course, if LOAD_HYBRID_BACKEND is disable, then the hybrid plugin will be not loaded.